Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre-alpine
+FROM openjdk:11-jre-alpine
```

Comment:
AFAIK this would not work, unfortunately, because of Java11 modularization. 
For now you can replace it with `FROM openjdk:11-jre-slim`.

Reply:
Thanks for the notice! Fixed the issue